README.md

```diff
@@ -172,21 +172,54 @@ app.listen(80, function () {
 })
 ```
 
+### CORS and CSRF
+
+The default configuration of this module implements the CORS specification as outlined by the W3C. That specification is designed to allow clients to make cross-origin requests, not to prevent servers from responding to requests. Thus, in the case of a CORS "failure", where the request's origin is not allowed, the browser will refuse to deliver the response _after_ the server responds as if the request had been valid.
```

Comment:
@dougwilson I tried to accurately capture your discussion from https://github.com/expressjs/cors/issues/109 here, please let me know if I misrepresented anything!